action.yml

```diff
@@ -14,11 +14,8 @@ inputs:
   access_secret:
     description: "access_secret"
     required: true
-  build1:
-    description: "build1"
-    required: true
-  build2:
-    description: "build2"
+  architecture:
+    description: "architecture"
```

Comment:
I like the approach on passing a single arch and using a build matrix to call it with different inputs. It adds to runtime, because each run will happen on a separate runner, but that is negligible (15s). On the plus side, this allows us to trigger or rerun a build for just one architecture.